### PR TITLE
Added the docker-compose

### PR DESCRIPTION
**Pull request by harsh4870**
Added **Docker compose** to quick setup stack with Postgress-pgvector along with Adminer to access DB with UI

You can start project stack by following steps

**Prerequisite**
- Update necessary vars in `docker-compose.yaml` from **FITBIT**

**Steps**

1. Start services - `./run-compose.sh`
2. Application accessible at - `http://localhost:8989`
3. Adminer accessible at - `http://localhost:8080`
4. Stop services - `./run-compose.sh down`

### CODE REVIEW

File: run-compose.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```
Comment (galeone) [inline run-compose.sh:1]: This should be ` #!/bin/env bash` to rely upon env to get the real location of bash

File: run-compose.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+docker pull pgvector/pgvector:pg16
```
Comment (galeone) [inline run-compose.sh:2]: I guess this is not needed since docker compose will download the image if not found

File: run-compose.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+mkdir -p volume
+
+if [ ! -d "$PWD/volume/postgresql" ]; then
+   mkdir -p volume/postgresql/data
+   chmod 755 -R $PWD/volume/postgresql/
+fi
+
+if [ -z $1 ]; then
+  chmod 755 -R $PWD/volume/
+  docker-compose -f docker-compose.yaml up -d
```
Comment (galeone) [inline run-compose.sh:13]: afaik docker-compose is deprecated. We should use `docker compose` (that's inside docker)

File: docker-compose.yaml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  db:
+    container_name: postgres
+    image: pgvector/pgvector:pg16
+    restart: always
+    hostname: postgres
+    shm_size: 128mb
+    volumes:
+      - $PWD/volume/postgresql/data:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
```
Comment (galeone) [inline docker-compose.yaml:13]: no need to map internal port to external port, this can be only:

`- "5432"`

File: docker-compose.yaml [hunk 5]
```diff
@@ -0,0 +1,94 @@
+version: "3.7"
```
Comment (galeone) [inline docker-compose.yaml:1]: with `docker compose` (the one that we should use) this line is deprecated. Just remove it

**Review by galeone — CHANGES_REQUESTED** [review]
Thank you very much for contributing!
I reviewed these changes and gave some suggestion about how to change it in order to use `docker compose` instead of `docker-compose` (deprecated), and some minor thing.

**Comment by harsh4870** [comment]
Thanks for suggstions, updated the changes

**Comment by galeone** [comment]
Merged - thank you very much for this PR! I'm going to update the README in the next PR adding a section on how to execute it through docker compose.

**Pull request merged**